File: Level2/3.124나라의숫자/3.124나라의숫자.cpp
#include <string>
#include <vector>
#include <algorithm>
using namespace std;

string solution(int n) {
    string answer = "";
    while(n!=0){
        if(n%3==0){
            answer +="4";
        }else if(n%3==1){
            answer +="1";
        }else{
            answer +="2";
        }
        n = (n-1)/3;
    }
    reverse(answer.begin(), answer.end());
    return answer;
}

File: Level2/3.124나라의숫자/3.124나라의숫자.py
def solution(n):
    answer = ''
    while n!=0:
        if n%3==0:
            answer += "4"
        elif n%3==1:
            answer += "1"
        else:
            answer += "2"
        n = int((n-1)/3)
    answer = answer[::-1]
    return answer